library/src/scala/quoted/Quotes.scala

```diff
@@ -3612,21 +3608,17 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
         def declaredMethods: List[Symbol]
 
         /** Get named non-private methods declared or inherited */
-        // TODO: deprecate in 3.1.0 and remove @experimental from methodMember
-        // @deprecated("Use methodMember", "3.1")
+        @deprecated("Use methodMember", "3.1.0")
         def memberMethod(name: String): List[Symbol]
 
         /** Get named non-private methods declared or inherited */
-        @experimental
         def methodMember(name: String): List[Symbol]
```

Comment:
Why do both methodMember and methodMembers return a List[Symbol] ? Shouldn't methodMember return a Symbol?

Reply:
Those are the overloads